src/common/helpers/responseHelper.ts

```diff
@@ -0,0 +1,70 @@
+import { Response } from "express";
+
+enum Status {
+    Error400 = 400,
```

Comment:
I recommend looking up what each of these error codes mean and using a friendly name for them. For example, instead of `Error400`, use `BadRequest`